README.md

```diff
@@ -528,6 +529,24 @@ sslv2, sslv3, tlsv1, sslv23, tlsv11, tlsv12
 The default value for the protocol if not set is tlsv12. The SSLConfig.setEnabled should be called explicitly to enable the SSL.
 The path of the certificate should be correctly provided. 
 
+# AWS Cloud Discovery
+
+Note: You need to provide compile flag -DHZ_BUILD_WITH_SSL when compiling since Aws depends on openssl library.
+
+The C++ client can discover the exiting Hazelcast servers in Amazon AWS environment. The client queries the Amazon AWS environment using the "describe-instances (http://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html)" query of AWS. The client only finds the up and running instances and filters them based on the filter config provided at the ClientAwsConfig configuration.
```

Comment:
...Hazelcast servers in the Amazon AWS.....The client finds only the up and.....